google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardMavenLibrary.java

```diff
@@ -0,0 +1,117 @@
+  JSTL("JSP Standard Tag Library (JSTL)", new RepositoryLibraryProperties(
+      "javax.servlet", "jstl", RepositoryUtils.LatestVersionId), DependencyScope.PROVIDED),
+  APP_ENGINE_API("App Engine API", new RepositoryLibraryProperties(
+      "com.google.appengine", "appengine-api-1.0-sdk", RepositoryUtils.LatestVersionId),
+      DependencyScope.COMPILE),
+  ENDPOINTS("App Engine Endpoints", new RepositoryLibraryProperties(
+      "com.google.appengine", "appengine-endpoints", RepositoryUtils.LatestVersionId),
+      DependencyScope.COMPILE),
+  OBJECTIFY("Objectify", new RepositoryLibraryProperties(
+      "com.googlecode.objectify", "objectify", RepositoryUtils.LatestVersionId),
+      DependencyScope.COMPILE);
+  // TODO: do we need these?
+  //  JSR_107_CACHE("JSR 107: Java Temporary Caching API", new RepositoryLibraryProperties(
+  //      "net.sf.jsr107cache", "jsr107cache", RepositoryUtils.LatestVersionId)),
+  //  APP_ENGINE_JSR_107_CACHE("JSR 107: Java Temporary Caching API",
+  //     new RepositoryLibraryProperties("com.google.appengine", "appengine",
+  //        RepositoryUtils.LatestVersionId));
+
+  private final String displayName;
+  private final RepositoryLibraryProperties libraryProperties;
+  private final DependencyScope scope;
+
+  AppEngineStandardMavenLibrary(String displayName, RepositoryLibraryProperties libraryProperties,
+      DependencyScope scope) {
+    this.displayName = displayName;
+    this.libraryProperties = libraryProperties;
+    this.scope = scope;
+  }
+
+  public String getDisplayName() {
+    return displayName;
+  }
+
+  public RepositoryLibraryProperties getLibraryProperties() {
+    return libraryProperties;
+  }
+
+  public DependencyScope getScope() {
+    return scope;
+  }
+
+  @Nullable
+  public static AppEngineStandardMavenLibrary getLibraryByDisplayName(final String name) {
+    return getLibrary(new Predicate<AppEngineStandardMavenLibrary>() {
+      @Override
+      public boolean apply(@Nullable AppEngineStandardMavenLibrary library) {
+        return library != null && name.equals(library.getDisplayName());
+      }
+    });
+  }
+
+  @Nullable
+  public static AppEngineStandardMavenLibrary getLibraryByMavenDisplayName(final String name) {
+    return getLibrary(new Predicate<AppEngineStandardMavenLibrary>() {
+      @Override
+      public boolean apply(@Nullable AppEngineStandardMavenLibrary library) {
+        return library != null
+            && name.equals(toMavenDisplayVersion(library.getLibraryProperties()));
+      }
+    });
+  }
+
+  public static AppEngineStandardMavenLibrary getLibrary(
+      Predicate<AppEngineStandardMavenLibrary> predicate) {
+    return Iterables.find(
+        Arrays.asList(AppEngineStandardMavenLibrary.values()),
+        predicate,
+        null /*default value*/);
+  }
+
+  /**
+   * Certain maven versions like "LATEST" are displayed differently - e.g. "Latest", so we need to
```

Comment:
Yea I wondered that too. Could be that the intellij maven plugin is using 2 internally. I'll check whats going on.